service/matching/ack_manager_test.go

```diff
@@ -28,35 +28,49 @@ import (
 	"math/rand"
 	"testing"
 
+	"github.com/golang/mock/gomock"
+
 	"github.com/stretchr/testify/require"
-	"go.temporal.io/server/common/log"
 )
 
 func TestAckManager_AddingTasksIncreasesBacklogCounter(t *testing.T) {
+	controller := gomock.NewController(t)
+	defer controller.Finish()
+	backlogMgr := newBacklogMgr(controller)
+
 	t.Parallel()
```

Comment:
I think t.Parallel() should always be the very first thing in any test where it appears